File: MyApplication/app/src/main/java/com/example/olivet/myapplication/Note.java
package com.example.olivet.myapplication;

/**
 * Created by fdubois on 16/01/18.
 */

public class Note {

    private int idNote;
    private int prototype;
    private int originalite;
    private int demarcheSI;
    private int pluriDisciplinarite;
    private int maitrise;
    private int devDurable;

    // Constructeur
    public Note(int idNote, int prototype, int originalite, int demarcheSI, int pluriDisciplinarite, int maitrise, int devDurable) {
        this.idNote=idNote;
        this.prototype=prototype;
        this.originalite=originalite;
        this.demarcheSI=demarcheSI;
        this.pluriDisciplinarite=pluriDisciplinarite;
        this.maitrise=maitrise;
        this.devDurable=devDurable;
    }

    public int getIdNote() {
        return idNote;
    }

    public void setIdNote(int id) {
        this.idNote = idNote;
    }

    public int getOriginalite() {
        return originalite;
    }

    public void setOriginalite(int originalite) {
        this.originalite = originalite;
    }

    public int getPrototype() {
        return prototype;
    }

    public void setPrototype(int prototype) {
        this.prototype = prototype;
    }

    public int getDemarcheSI() {
        return demarcheSI;
    }

    public void setDemarcheSI(int demarcheSI) {
        this.demarcheSI = demarcheSI;
    }

    public int getPluriDisciplinarite() {
        return pluriDisciplinarite;
    }

    public void setPluriDisciplinarite(int pluriDisciplinarite) {
        this.pluriDisciplinarite = pluriDisciplinarite;
    }

    public int getMaitrise() {
        return maitrise;
    }

    public void setMaitrise(int maitrise) {
        this.maitrise = maitrise;
    }

    public int getDevDurable() {
        return devDurable;
    }

    public void setDevDurable(int devDurable) {
        this.devDurable = devDurable;
    }
}

File: BD/Classe/Note.php
<?php
/**
 *
 */
class Note
{

  private $_id;
  private $_prototype;
  private $_originalite;
  private $_demarcheSI;
  private $_pluriDisciplinarite;
  private $_maitrise;
  private $_devDurable;

  function __construct($id,$proto,$originalite,$demarcheSI,$pluriDisciplinarite,$maitrise,$devDurable)
  {
    $this->_id = $id;
    $this->_prototype = $proto;
    $this->_originalite = $originalite;
    $this->_demarcheSI = $demarcheSI;
    $this->_pluriDisciplinarite = $pluriDisciplinarite;
    $this->_maitrise = $maitrise;
    $this->_devDurable = $devDurable;
  }

  public function getID()
  {
    return $this->_id;
  }

  public function getProto()
  {
    return $this->_prototype;
  }

  public function getOrigin()
  {
    return $this->_originalite;
  }

  public function getdemarcheSI()
  {
    return $this->_demarcheSI;
  }

  public function getPluriDiscipline()
  {
    return $this->_pluriDisciplinarite;
  }

  public function getMaitrise()
  {
    return $this->_maitrise;
  }

  public function getDurable()
  {
    return $this->_devDurable;
  }

  public function setProto($proto)
  {
    $this->_prototype = $proto;
  }

  public function setOrigin($origin)
  {
    $this->_originalite= $origin;
  }

  public function setdemarcheSI($demarcheSI)
  {
    $this->_demarcheSI = $demarcheSI;
  }

  public function setPluriDiscipline($pluriDisciplinarite)
  {
    $this->_pluriDisciplinarite = $pluriDisciplinarite;
  }

  public function setMaitrise($maitrise)
  {
    $this->_maitrise = $maitrise;
  }

  public function setDurable($devDurable)
  {
    $this->_devDurable = $devDurable;
  }

}



 ?>
